public/operationsgateway-settings.example.json

```diff
@@ -1,4 +1,5 @@
 {
+  "facilityName": "Generic",
```

Comment:
Do we need facility name? This was needed for `ICAT` as a lot of features (especially in TopCAT backend) required knowing the facility name. But if our backend doesn't need it we should remove it

Reply:
Oh yeah that can go